contentcuration/contentcuration/tests/base.py

```diff
@@ -42,6 +45,8 @@ class StudioTestCase(TestCase, BucketTestMixin):
     def setUpClass(cls):
         super(StudioTestCase, cls).setUpClass()
         call_command('loadconstants')
+        cls.url = "http://127.0.0.1:8000"
```

Comment:
maybe this come from the `SITES` variable in `loadconstants.py`?  in case someone ever decides to change it?